rs/nns/propose_to_make_neuron_27_a_known_neuron

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+GOVERNANCE=rrkah-fqaaa-aaaaa-aaaaq-cai
+
+dfx --identity hsm \
+    canister \
+    --network ic \
+    call \
+    --candid rs/nns/governance/canister/governance.did \
+    $GOVERNANCE \
+    manage_neuron \
+    '(record {
+        neuron_id_or_subaccount = opt variant { NeuronId = record { id = 51 }} ;
+        command = opt variant { MakeProposal = record {
+            title = opt "Make Neuron 27 A Known Neuron";
+            url = "https://forum.dfinity.org/t/strengthening-governance-by-aligning-neuron-28s-voting-power-under-neuron-27/35957";
+            summary = "# Make Neuron 27 a Known Neuron
+
+Its name will be “DFINITY Foundation”. Its description will be as quoted in Appendix A.
+
+# Background
+
+## Current Status of Neuron 27
+
+In the NNS Dapp and the IC Dashboard, this neuron is displayed as though it is
+a known neuron named “DFINITY Foundation”. For example, when you view a proposal
+in IC Dashboard, towards the bottom of the page, it shows the votes of known
+neurons, as well as the vote of this neuron. However, this neuron was never
+officially registered as a known neuron in the NNS.
+
+## Why This Neuron Was Not a Known Neuron Before
+
+The known neuron feature did not exist in early versions of NNS. By the time the
+known neuron feature was added, the NNS Dapp and IC Dashboard were already
+treating this neuron specially.
+
+## What Changed
+
+The development of the “Public and Private Neurons” feature (approved in
+[motion proposal 130832]) is nearing completion (per the
+[plan described in the forum]). The main remaining step is to begin enforcing
+private visibility.
+
+[motion proposal 130832]: https://dashboard.internetcomputer.org/proposal/130832
+[plan described in the forum]: https://forum.dfinity.org/t/request-for-comments-api-changes-for-public-private-neurons/33360
+
+However, if private enforcement were to be enabled now, neuron 27'"'"'s votes
+would become invisible. This proposal avoids that, because when private
+enforcement is enabled, the votes of known neurons (and public neurons) will
+(still) be visible.
+
+# Impact
+
+This removes one of the last blockers for the full release of the “Public and
+Private Neurons” feature.
+
+This ensures that the current status quo is maintained. I.e., this neuron’s
+votes will remain visible after the full release of the “Public and Private
+Neurons” feature.
+
+# Appendix A: Description
```

Comment:
Maybe "Known neuron's description" or so? I am not sure it if it is clear to readers here what "description" refers to. ( I know you mentioned it on top but this i already a bit away so people might not remember here)

Reply:
Done.